docs/user-guide/leverage-cli/reference/kubectl.md

```diff
@@ -0,0 +1,52 @@
+# Command: `kubectl`
+
+!!! info "Regarding Leverage Toolbox versions"
+    For using this feature Leverage Toolbox versions `1.2.7-0.1.7` and up, or `1.3.5-0.1.7` and up must be used.
```

Comment:
```suggestion
    To have this feature available, Leverage Toolbox versions `1.2.7-0.1.7` and up, or `1.3.5-0.1.7` and up must be used.
```
Just to avoid the repetition of "use".